_database_api/maintenance.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Maintenance
+layout: default
+---
+
+# Maintenance
+
+--- row ---
+
+**Maintenance attributes**
+
+{:.table}
+| field       | type   | description                      |
+| ----------  | ------ | -------------------------------- |
+| id          | string | unique ID                        |
+| database_id | string | database identifier              |
+| type        | string | type of the maintenance          |
+| status      | string | status of the maintenance        |
+| started_at  | date   | when the maintenance started     |
+| ended_at    | date   | when the maintenance ended       |
+
+||| col |||
+
+Example object:
+
+```json
+{
+  "maintenance": {
+    "id": "my-maintenance-123",
+    "database_id": "my-db-123",
+    "type": "no-op",
+    "status": "done",
+    "started_at": "2023-07-04T15:59:50.407Z",
+    "ended_at": "2023-07-04T15:59:50.408Z"
+  }
+}
+```
+--- row ---
+
+## List database Maintenance
+
+--- row ---
+
+`GET https://$DB_API_URL/api/databases/[:db_id]/maintenances?page=[:page_number]&per_page=[:per_page_number]`
```

Comment:
We don't usually display the pagination parameter. Instead, you can do the same as for the logs-archive, for example. By specifying that this request supports the pagination system.